Packs/AtlassianConfluenceCloud/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Atlassian Confluence Cloud
+
+Added the ability to fetch events in Cortex XSIAM.
```

Comment:
check with @yaakovpraisler how to release only for xsiam and not xsoar - this version shouldnt be showing up in the xsoar marketplace

Reply:
done

Reply:
@talzich Please add the XSIAM tags as discussed, as this content item belongs to XSOAR too.

Reply:
```suggestion
<~XSIAM>
Added the ability to fetch events in Cortex XSIAM.
</~XSIAM>
```

Reply:
@yaakovpraisler is this part of update release notes? wonder if might be worth it to add an interactive question for which marketplace a release is for so more people are familiar with this feature

Reply:
@JudahSchwartz An interactive question is a good idea and should work 👍